app/models/agents/webhook_agent.rb

```diff
@@ -84,7 +85,7 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [response_message, 201]
+      [response_message, (interpolated['code'] || '201').to_i]
```

Comment:
I think that needs to be `interpolated['code'].presence` otherwise it will return `""` when `code` is not configured. You could also use `210` as an integer to avoid the type cast.